### PR TITLE
Trim up docs home page, add date-meta, add readme

### PR DESCRIPTION
**Pull request by jthomasmock**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Some docs improvements:
- Trim up docs home page
- Add date-meta to all pages for tracking changes
- Add readme for building locally
- Remove the "Open Issue" option until Quarto builds a "choose issue type", changed to edit/view source
- Changed various links from rstudio.com to posit.co
- Corrected misspelling of `builtType` to `buildType`

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

### CODE REVIEW

File: docs/user/rstudio/index.qmd [hunk 1]
```diff
@@ -27,20 +29,19 @@ The following documentation helps a user understand the core workflows in RStudi
 
 ## Commercial Edition
 
-Posit Workbench has several features that differentiate it from the open-source [RStudio desktop](https://www.rstudio.com/products/rstudio/) and [RStudio Server](https://www.rstudio.com/products/rstudio/#rstudio-server), which you can read about on the [Workbench comparison page](https://www.rstudio.com/products/workbench/comparison/). The main advantages of using Posit Workbench are:
+Posit Workbench has several features that differentiate it from the open-source [RStudio desktop](https://posit.co/products/open-source/rstudio/) and [RStudio Server](https://posit.co/products/open-source/rstudio-server/), which you can read about on the [Workbench comparison page](https://posit.co/products/enterprise/workbench/). The main advantages of using Posit Workbench are:
```
Comment (ronblum) [inline docs/user/rstudio/index.qmd:32]: Change
    `Workbench comparison page`
to
    `PositWorkbench comparison page`

File: docs/user/rstudio/index.qmd [hunk 2]
```diff
@@ -49,16 +50,19 @@ Posit Workbench administrators can reference the [Posit Workbench Administrator
...
-- [RStudio Desktop download](https://www.rstudio.com/products/rstudio/download/) - available for Windows/Mac/Linux
-- [RStudio Server download](https://www.rstudio.com/products/rstudio/download-server/) - available for Linux
+- [RStudio Desktop download](https://posit.co/downloads/) - available for Windows/Mac/Linux
+- [RStudio Server download](https://posit.co/downloads/) - available for Linux
 
 ### Commercial
 
 - [Posit Workbench](https://docs.rstudio.com/rsw/installation/) - administration guide for installation and configuration
-- [RStudio Desktop Pro](https://www.rstudio.com/products/rstudio-desktop-pro/download-commercial/) - available for Windows/Mac/Linux
+- [RStudio Desktop Pro](https://posit.co/downloads/) - available for Windows/Mac/Linux
```
Comment (ronblum) [inline docs/user/rstudio/index.qmd:59]: Should the link here be `https://posit.co/downloads/` or `https://posit.co/download/rstudio-desktop-pro/` ?

File: docs/user/rstudio/readme.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# RStudio User Guide
+
+-   Source code: <https://github.com/rstudio/rstudio/tree/main/docs/user/rstudio>
+-   Public URL: <https://docs.rstudio.com/ide/user/>
+
+> If you’re interested in getting involved with open source, contributing to documentation is a good—and helpful—place to start. - [James Turnbill](https://increment.com/documentation/documentation-as-a-gateway-to-open-source/)
+
+If you have found issues with the RStudio User Guide, please check for existing duplicate issues/pull requests. 
+If a matching Issue/Pull Request doesn't exist, please then open an [Issue](https://github.com/rstudio/rstudio/issues/new/choose) or Pull Request.
```
Comment (ronblum) [inline docs/user/rstudio/readme.md:9]: Is "Pull Request" intended to be a link, such as `https://github.com/rstudio/rstudio/pulls` ?

File: docs/user/rstudio/readme.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Alternatively, selecting "Edit this Page" will prepare you to open a quick Pull Request from your browser.
+
+## Development
+
+To locally test changes to the User Guide, you will need to do the following:
+
+1.  [Install Quarto](https://quarto.org/docs/getting-started/installation.html) or RStudio 2022.07.2 or later, which bundles Quarto.
+
+-   Download RStudio from [posit.co/downloads/](https://posit.co/downloads/)
+-   Download the CLI from [github.com/quarto-dev](https://github.com/quarto-dev/quarto-cli/releases/latest) for your operating system
```
Comment (ronblum) [inline docs/user/rstudio/readme.md:20]: What's the reason for including this link when we've given instructions for downloading the CLI in line 17, `1. [Install Quarto]` etc.?

File: docs/user/rstudio/readme.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+cd docs/user/rstudio
+quarto preview
+```
+
+-   From the R console, enter `quarto::quarto_preview()` on the file you've changed like below:
+
+```r
+quarto::quarto_preview(file = "docs/user/rstudio/index.qmd")
+```
+
```
Comment (ronblum) [inline docs/user/rstudio/readme.md:43]: Should we mention that the following warnings are expected?
```
WARNING: Unknown var buildType specified in a var shortcode.
WARNING: Unknown var version specified in a var shortcode.
```

**Review by ronblum — COMMENTED** [review]
LGEFAFCAQ (looks good except for a few comments and questions).

File: docs/user/rstudio/index.qmd (hunk 1 above)
Comment (jthomasmock) [reply docs/user/rstudio/index.qmd:32]: Changed

File: docs/user/rstudio/readme.md (hunk 3 above)
Comment (jthomasmock) [reply docs/user/rstudio/readme.md:9]: The PR workflow would be via "Edit this page" or a more manual process as opposed to the direct link to an issue (I wanted to very very specific link to the "choose" an issue type).

**Review by ronblum — APPROVED** [review]

**Pull request merged**